File: Custom_responses_that_don’t_hit_origin_servers/nodejs/ignore_post_and_put_requests.js
async function f(event) {
    const request = event.request;

    // POST/PUT 返回 403
    if (request.method === 'POST' || request.method === 'PUT') {
        return {status: 403};
    } else {
    	// 其他继续回源
        return request;
    }
}

exports.handler = f;

File: Custom_responses_that_don’t_hit_origin_servers/lua/ignore_post_and_put_requests.lua
local _M = {}

function _M.handler(event)
    local request = event.request

    -- POST/PUT 返回 403
    if request.method == 'POST' or request.method == 'PUT' then
        return {status = 403}
    end

    -- 其他继续回源
    return event.request
end

return _M